postman/collection-dir/cybersource/Flow Testcases/Happy Cases/Scenario13 Incremental auth/Payments-Create Copy/request.json

```diff
@@ -0,0 +1,164 @@
+{
+  "auth": {
+    "type": "apikey",
+    "apikey": [
+      {
+        "key": "value",
+        "value": "{{api_key}}",
+        "type": "string"
+      },
+      {
+        "key": "key",
+        "value": "api-key",
+        "type": "string"
+      },
+      {
+        "key": "in",
+        "value": "header",
+        "type": "string"
+      }
+    ]
+  },
+  "method": "POST",
+  "header": [
+    {
+      "key": "Content-Type",
+      "value": "application/json"
+    },
+    {
+      "key": "Accept",
+      "value": "application/json"
+    },
+    {
+      "key": "x-feature",
+      "value": "router-custom",
+      "type": "text",
+      "disabled": true
+    },
+    {
+      "key": "Authorization",
+      "value": "",
+      "type": "text",
+      "disabled": true
+    },
+    {
+      "key": "x-feature",
+      "value": "integ-custom",
+      "type": "text",
+      "disabled": true
+    }
+  ],
+  "body": {
+    "mode": "raw",
+    "options": {
+      "raw": {
+        "language": "json"
+      }
+    },
+    "raw_json_formatted": {
+      "amount": 1000,
+      "currency": "USD",
+      "confirm": true,
+      "capture_method": "manual",
+      "customer_id": "cus_wKqxfVdHDDLWuiM144",
```

Comment:
We've have `"{{$customer_id}}"` and I believe we can use that. But also you might want to update it else where as well.